crypto-conditions/src/main/java/org/interledger/cryptoconditions/CryptoConditionReader.java

```diff
@@ -34,9 +35,7 @@
    * Reads a DER encoded condition from the buffer.
    *
    * @param buffer contains the raw DER encoded condition.
-   *
```

Comment:
Don't think these newlines are correct. See my comment below in `InterledgerPacket`.